scripts/release-hooks.mjs

```
@@ -96,6 +111,7 @@ if (process.env.RELEASE_IT_DRY_RUN === "1") {
   for (const { rel } of packageJsonFiles) console.log(`    - ${rel}`);
   console.log(`  [dry-run] Would bump ${manifestFiles.length} manifest.json files to version ${manifestVersion}:`);
   for (const { rel } of manifestFiles) console.log(`    - ${rel}`);
+  console.log(`  [dry-run] Changelog: ${changelogStamp.reason}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)